src/Blocks/BlockHandler.h

```diff
@@ -110,8 +110,11 @@ class cBlockHandler
 
 	/** Checks if the player can build "inside" this block.
 	For example blocks placed "on" snow will be placed at the same position. So: Snow ignores Build collision
+	@param a_Pos Position of the block
+	@param a_Player Player trying to build on the block
+	@param a_Meta Meta value of the block
```

Comment:
Okay, that's news to me. Reading through some discussions on the topic, I think this is pointless anyway, so let's just keep it as it is.